### ISSUE TITLE
Omovertebral bone

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10859]
**Preferred term label:**
Omovertebral bone

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal osseous connection between the superomedial border of the scapula to a spinous process in the cervical spine

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sprengel deformity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001177
- label: Omovertebral bone
- parents: [Abnormal scapula morphology (HP:0000782)](https://hpo.jax.org/app/browse/term/HP:0000782)
- pmid: PMID:38908946
- def: An osseous connection between the superomedial border of the right scapula and C5-C6 spinous processes.
- comment: The omovertebral bone is seen in approximately one-third of patients with Sprengel deformity. The clinical presentation of SD varies, ranging from simply limited shoulder abduction and mild cosmetic deformity to severe clinical abnormalities. Nonsurgical management is appropriate for mild cases, while severe deformities require surgery. See Figure 1 of PMID:38908946.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10859
- synonyms: Os omovertebrale(NONE)
